modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/ImageField.testcase

```diff
@@ -149,12 +149,29 @@ definition {
 	}
 
 	@description = "This is a test for LPS-98073. As a Developer, I want to Implement the image field type"
-	@ignore = "true"
 	@priority = "3"
 	test SetImageFieldAsRepeatable {
 		property portal.acceptance = "false";
 
-		// TODO LPS-98073 SetImageFieldAsRepeatable pending implementation
+		var structureTitle = "My Structure";
+		var newLabel = "Title";
+		var webTitle = "WC Title";
+
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
+
+		NavItem.gotoStructures();
+
+		WebContentStructures.addCP(
+			structureDescription = "I'm a Structure",
+			structureName = "${structureTitle}");
+
+		DataEngine.addField(
+			fieldFieldLabel = "${newLabel}",
+			fieldName = "Image");
+
+		DataEngine.editFieldRepeatable(fieldFieldLabel = "${newLabel}");
+
+		DataEngine.validateDuplicatedButton(key_fieldName = "${newLabel}");
```

Comment:
hey @gleicelisbino , could you please validate the edition persistence?
In this case, you set the field as repeatable and then save the structure.
Reopen the structure and assert it is still set as repeatable.